CHANGELOG.md

```diff
@@ -61,6 +61,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 - **Pulsar Scaler**: Improve error messages for unsuccessful connections ([#4563](https://github.com/kedacore/keda/issues/4563))
 - **Security:** Enable secret scanning in GitHub repo
 - **RabbitMQ Scaler**: Add support for `unsafeSsl` in trigger metadata ([#4448](https://github.com/kedacore/keda/issues/4448))
+- **Prometheus Metrics**: Add new metric with Keda build info ([#4576](https://github.com/kedacore/keda/pull/4576))
```

Comment:
```suggestion
- **Prometheus Metrics**: Add new metric with KEDA build info ([#4576](https://github.com/kedacore/keda/pull/4576))
```

We should link an issue here, not the PR. Could you please create one and link it?